db/dexie_implementation.go

```diff
@@ -241,6 +241,22 @@ func (db *DB) AddMiniHeaders(miniHeaders []*types.MiniHeader) (added []*types.Mi
 	return dexietypes.MiniHeadersToCommonType(jsAdded), dexietypes.MiniHeadersToCommonType(jsRemoved), nil
 }
 
+func (db *DB) ResetMiniHeaders(miniHeaders []*types.MiniHeader) (added []*types.MiniHeader, removed []*types.MiniHeader, err error) {
```

Comment:
I think we should add a comment explaining what `ResetMiniHeaders` does since it is less obvious than something like `DeleteOrder`. (Honestly, I should have done the same for other non-obvious methods like `UpdateOrder`).

This should also be added to the `Database` interface, which helps ensure that both the Dexie implementation and the SQL implementation have the exact same methods.